src/js/components/DataTable/DataTable.js

```diff
@@ -206,6 +208,19 @@ const DataTable = ({
     if (nextScrollOffset !== scrollOffset) setScrollOffset(nextScrollOffset);
   });
 
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  useLayoutEffect(() => {
+    if (placeholder && placeholderRef.current) {
+      const nextHeaderHeight = headerRef.current
+        ? headerRef.current.getBoundingClientRect().height : 0;
+      const nextFooterHeight = footerRef.current
+        ? footerRef.current.getBoundingClientRect().height : 0;
+      const nextMinHeight = nextHeaderHeight + nextFooterHeight +
```

Comment:
Why do we include the header and footer heights here but then also pass them as `top` and `bottom` to StyledPlaceholder? As with my other comment for the other layout effect, I'm wondering if we should render the StyledPlaceholder and then have a single layout effect that sets top, bottom, and min-height directly on the `placeholderRef.current`

Reply:
I'll look at that. Really we're wanting this element to be relative to the body and without a resize handler I think we're better off making this parented to the body, but I think that means it must be in a `<tr><td>`